docs/user-guides/capacity-plannig.md

```diff
@@ -0,0 +1,173 @@
+
+**Pod priority**
+
+Pod priority has the integer value, and the higher value, the higher priority.
+
+Each Vald component has the default priority value:
+
+- Agent: 1000000000
+- Discoverer: 1000000
+- Filter Gateway: 1000000
+- LB Gateway: 1000000
+- Index Manager: 1000000
+
+Therefore, the order of priority is as follows:
+
+```bash
+Agent > Discoverer = Filter Gateway = LB Gateway = Index Manger
+```
+
+Those values will be helpful when the Pods other than the Vald component are in the same Node.
+
+It is easy to change by editing your `values.yaml`.
+
+```yaml
+# e.g. LB Gateway podPriority settings.
+...
+  gateway:
+    lb:
+    ...
+      podPriority:
+        enabled: true
+        value: {new values}
+      ...
+```
+
+**QoS**
+
+QoS value can be either Guaranteed, Burstable, or BestEffort.
+And, QoS priority is higher in the order of Guaranteed, Burstable, BestEffort, and Kubernetes will kill Pods in ascending order of importance.
+
+Resource request and limit determine QoS.
+
+The below table shows the condition for each QoS.
+
+|    QoS     |   request CPU   | request Memory |    limit CPU    | request Memory  | Sup.                                |
+| :--------: | :-------------: | :------------: | :-------------: | :-------------: | :---------------------------------- |
+| Guaranteed |       :o:       |      :o:       |       :o:       |       :o:       | All settings are required.          |
+| Guaranteed | :o: (:warning:) | :o: (:warning) | :o: (:warning:) | :o: (:warning:) | One to three settings are required. |
+| BestEffort |       :x:       |      :x:       |       :x:       |       :x:       | No setting is required.             |
+
+Vald requires many RAM resources because of on-memory indexing, so we highly recommend that you do not specify a limit, especially for the Vald Agent.
+In this case, QoS will be Burstable.
+
+**Throttling**
+
+The CPU throttling affects the pod performance.
+
+If it occurs, the Vald cluster operator must consider each component's CPU resource request and limit.
+It is easy to change by editing the `values.yaml` file and applying it.
+
+```yaml
+# e.g. LB Gateway resources settings.
+...
+  gateway:
+    lb:
+    ...
+      resources:
+        requests:
+          cpu: 200m
+          memory: 150Mi
+        limits:
+          cpu: 2000m
+          memory: 700Mi
+    ...
+```
+
+<div class="warning">
+Please take care of pod priority and QoS.
+</div>
+
+### Node & Pod affinity
+Kubernetes scheduler often applies Pods on Node based on resource availability.
+
+In production usage, other components sometimes work on the Kubernetes cluster where the Vald cluster runs.
+Depending on the situation, you may want to deploy to a different Node: e.g., when running a machine learning component that requires high memory on an independent Node.
+
+In this situation, we recommend you to set the affinity/anti-affinity configuration for each Vald component.
+It is easy to change by editing your `values.yaml`.
```

Comment:
In addition we should describe about helm default antiAffinity, that locates Agent Pod and should not edit these configuration.